Added replication limitation to README (#45)
dolthub/doltgresql: README.md

# Limitations

> * No Git-style CLI for version control, only a SQL interface (compared to [Dolt](https://github.com/dolthub/dolt)).

Reviewer: ```suggestion
* No [Git-style CLI](https://docs.dolthub.com/cli-reference/cli) for version control like in [Dolt](https://github.com/dolthub/dolt), only a SQL interface.
```
